Author: Guillaume Horel <guillaume.horel@gmail.com>
URL: https://aur.archlinux.org/packages/python-ecos/
Summary: Allows us to compile ecos as a seperate package and install.
----
diff -urN ecos-python-2.0.7rc2-orig/setup.py ecos-python-2.0.7rc2/setup.py
--- ecos-python-2.0.7rc2-orig/setup.py	2018-05-23 22:16:42.967677526 -0400
+++ ecos-python-2.0.7rc2/setup.py	2018-05-23 22:17:41.981221901 -0400
@@ -9,7 +9,7 @@
 from glob import glob
 from platform import system
 
-lib = []
+lib = ['ecos']
 if system() == 'Linux':
     lib += ['rt']
 
@@ -21,25 +21,8 @@
                         ('DLONG', None),
                         ('LDL_LONG', None),
                         ('CTRLC', 1)],
-                    include_dirs = ['ecos/include',
-                        'ecos/external/amd/include',
-                        'ecos/external/ldl/include',
-                        'ecos/external/SuiteSparse_config'],
-                    sources = ['src/ecosmodule.c',
-                        'ecos/external/ldl/src/ldl.c',
-                        'ecos/src/cone.c',
-                        'ecos/src/ctrlc.c',
-                        'ecos/src/ecos.c',
-                        'ecos/src/equil.c',
-                        'ecos/src/expcone.c',
-                        'ecos/src/kkt.c',
-                        'ecos/src/preproc.c',
-                        'ecos/src/spla.c',
-                        'ecos/src/splamm.c',
-                        'ecos/src/timer.c',
-                        'ecos/src/wright_omega.c'
-                    ] + glob('ecos/external/amd/src/*.c')
-                      + glob('ecos/ecos_bb/*.c'))       # glob bb source files
+                    include_dirs = ['/usr/include/ecos'],
+                    sources = ['src/ecosmodule.c'])
 
 def set_builtin(name, value):
     if isinstance(__builtins__, dict):
